### ISSUE TITLE
15-1장. 코드 예상가격15-1장. 코드 예상가격

### ISSUE THREAD

**Issue by YunaOh84** [gilbutITbook/080228#13]
책에 있는 코드 그대로 입력한 것 같은데
예상 가격이 6.342로 똑같이 나옵니다. 
제가 오타가 있는 건지 한번 확인 부탁드립니다.

from keras.models import Sequential
from keras.layers import Dense
from sklearn.model_selection import train_test_split

import numpy
import pandas as pd
import tensorflow as tf

seed = 0
numpy.random.seed(seed)
tf.random.set_seed(3)

df = pd.read_csv('housing.csv', delim_whitespace=True, header=None)

dataset = df.values
X = dataset[:,0:13]
Y = dataset[:,13]

X_train, X_test, Y_train, Y_test = train_test_split(X, Y, test_size=0.3, random_state=seed)

model = Sequential()
model.add(Dense(30, input_dim=13, activation='relu'))
model.add(Dense(6, activation='relu'))
model.add(Dense(1))

model.compile(loss='mean_squared_error',
              optimizer='adam')

model.fit(X_train, Y_train, epochs=200, batch_size=10)

Y_prediction = model.predict(X_test).flatten()
for i in range(10):
    label = Y_test[i]
    prediction = Y_prediction[i]
    print("실제가격: {:.3f}, 예상가격: {:.3f}".format(label, prediction))

**Comment by waff1e** [comment]
python==3.7.9
tensorflow==2.0.0
keras==2.3.0
위 와 같은 환경에서 실행하면 예상가격이 정상적으로 나옵니다.

**Comment by stableh** [comment]
model.add(Dense(6, activation='relu')) 이 부분에서 노드를 6 보다 큰 숫자로 (저는 24으로 했습니다.) 설정하니 정상적으로 출력됩니다.
6으로 설정했을 시에 예상 값이 전부 6.342로 통일 되어버리는데 이건 아직 배움이 부족해 잘 모르겠습니다.
